packages/mui-lab/src/TimePicker/TimePicker.tsx

```diff
@@ -94,6 +94,10 @@ TimePicker.propTypes /* remove-proptypes */ = {
    * @default false
    */
   ampmInClock: PropTypes.bool,
+  /**
+   * @ignore
+   */
+  autoFocus: PropTypes.bool,
```

Comment:
Comments are missing here, make sure you complete the tasks below.

https://github.com/mui-org/material-ui/blob/master/CONTRIBUTING.md#updating-the-component-api-documentation

Reply:
> Comments are missing here, make sure you complete the tasks below.
> 
> https://github.com/mui-org/material-ui/blob/master/CONTRIBUTING.md#updating-the-component-api-documentation

I made a mistake. The comments are actually ignored by default in the relevant components.